tests/network/upgrade/conftest.py

```diff
@@ -72,12 +72,16 @@ def vmb_upgrade_mac_spoof(worker_node1, unprivileged_client, upgrade_linux_macsp
 
 @pytest.fixture(scope="session")
 def running_vma_upgrade_mac_spoof(vma_upgrade_mac_spoof):
-    return running_vm(vm=vma_upgrade_mac_spoof)
+    vma_upgrade_mac_spoof.start(wait=True)
```

Comment:
`vma_upgrade_mac_spoof` is already started upon creation (because it sets `runStrategy: Always`), so you shouldn't start it here as well.

Reply:
Done